src/components/VFRChartsList.js

```diff
@@ -54,9 +57,9 @@ class VFRChartsList extends Component {
 
 const styles = StyleSheet.create({
   container: {
-    alignItems: 'center',
-    backgroundColor: Colors.secondary,
     flex: 1,
+    alignItems: 'stretch',
+    backgroundColor: 'green',
```

Comment:
I tested it and if we don’t set the color of this view, you can see the menu underneath this view haha